keyboards/keebzdotnet/fme/readme.md

```diff
@@ -0,0 +1,15 @@
+# F-Me Macropad
+
+![F Me macropad](https://i.imgur.com/FKD43mB.jpg)
+
+A macropad designed after a meme board seen on Reddit.
+
+* Keyboard Maintainer: [keebzdotnet](https://github.com/keebzdotnet)
+* Hardware Supported: F-Me Macropad
+* Hardware Availability: https://www.keebz.net/shop/p/f-me
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader, to flash new firmware?

Reply:
@drashna I've added instructions for the bootloader :)